colorbleed/plugins/maya/load/load_yeti_cache.py

```diff
@@ -84,15 +84,26 @@ def remove(self, container):
     def update(self, container, representation):
 
         path = api.get_representation_path(representation)
+        namespace = "{}:".format(container["namespace"])
         members = cmds.sets(container['objectName'], query=True)
-        yeti_node = cmds.ls(members, type="pgYetiMaya", long=True)
+        yeti_node = cmds.ls(members, type="pgYetiMaya")
+
+        # TODO: Count the amount of nodes cached
+        # To ensure new nodes get created or old nodes get destroyed
 
         for node in yeti_node:
-            node_name = node.split(":")[-1]
+            # Remove local given namespace
+            node_name = node.split(namespace, 1)[-1]
+            node_name = node_name.replace(":", "_")
```

Comment:
What `:` will remain present after the namespace was removed? What is this replacing?

Also, what's the expected idea if magically the namespace does not match? Do we need to assert? (just wondering).

Reply:
When a cache is generated in a shot the Yeti node's full name is `character_x_01_:body_hairShape`  for example. When using the <NAME> token will take the namespace and node name as the full node name. When loading those caches in to the shot the name will be `shot0010:character_x_01_:body_hairShape`. Hence the split and replace, it is a quirk I will need to figure out.

The namespace is based on the namespace from the container. An assertion is possible but I do not see any danger in not asserting at the moment.